update_changelog.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+version=$(date +'v%Y%m%d-stable')
+
+#token to request pr informations
+token=$1
+if [[ -z "$token" ]];then
+    echo "Could not detect token parameter version :("
+    exit 1
+fi
+
+
+generate_changelog () {
+    local path=$1
+    local title="$2"
+    local fromCommit="$3"
+    local toCommit="$4"
+
+    echo "### $title"
+    echo ""
+    git -C $path log  --pretty="%s" --grep="Merge pull request #" $fromCommit.. | while read -r line ; do
```

Comment:
This assumes we never alter(ed) the default commit description when merging on Github. But I see no other option. Using `--merges` would also give us updates (merges of master into a feature branch).

But is there a reason you don't use the upper limit of the commit range here?
```suggestion
    git -C $path log  --pretty="%s" --grep="Merge pull request #" $fromCommit..$toCommit | while read -r line ; do
```